manifests/q/qBittorrent/qBittorrent/4.6.4/qBittorrent.qBittorrent.locale.en-US.yaml

```diff
@@ -0,0 +1,43 @@
+# Created with YamlCreate.ps1 v2.3.4 Dumplings Mod $debug=QUSU.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: qBittorrent.qBittorrent
+PackageVersion: 4.6.4
+PackageLocale: en-US
+Publisher: The qBittorrent project
+PublisherUrl: https://www.qbittorrent.org/
+PublisherSupportUrl: https://github.com/qbittorrent/qBittorrent/issues
+# PrivacyUrl:
+Author: |-
+  Sledgehammer999 <sledgehammer999@qbittorrent.org>
+  Christophe Dumez <chris@qbittorrent.org>
+PackageName: qBittorrent
+PackageUrl: https://www.qbittorrent.org/
+License: GPL-3.0
+LicenseUrl: https://github.com/qbittorrent/qBittorrent/blob/master/COPYING
+Copyright: Copyright © 2006-2024 The qBittorrent project
+# CopyrightUrl:
+ShortDescription: Free and reliable P2P Bittorent client.
+Description: |-
+  qBittorrent is a bittorrent client programmed in C++ / Qt that uses libtorrent (sometimes called libtorrent-rasterbar) by Arvid Norberg.
+  It aims to be a good alternative to all other bittorrent clients out there. qBittorrent is fast, stable and provides unicode support as well as many features.
+# Moniker:
+Tags:
+- bittorrent
+- bt
+- download
+- magnet
+- p2p
+- share
+- sharing
+- torrent
+- upload
+# ReleaseNotes:

```

Comment:
v4.6.4 changelog:
- BUGFIX: Correctly adjust "Add New torrent" dialog position in all the cases (glassez)
- BUGFIX: Change "metadata received" stop condition behavior (glassez)
- BUGFIX: Add a small delay before processing the key input of search boxes (Chocobo1)
- BUGFIX: Ensure the profile path is pointing to a directory (Chocobo1)
- RSS: Use better icons for RSS articles (glassez)
- WINDOWS: NSIS: Update French, Hungarian translations (MarcDrieu, foxi69)
- LINUX: Fix sorting when ICU isn't used (Chocobo1)
- LINUX: Fix invisible tray icon on Plasma 6 (tehcneko)

Reply:
Nah I'm waiting for the one on https://www.qbittorrent.org/news

Reply:
![image](https://github.com/microsoft/winget-pkgs/assets/159099382/d976f6df-5b68-4744-9d25-6068842b04e2)